applications/FluidDynamicsApplication/tests/test_FluidDynamicsApplication.py

```diff
@@ -106,7 +107,7 @@ def AssembleTestSuites():
     nightSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([IntegrationPointStatisticsTest]))
     nightSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([CFLOutputProcessTest]))
     nightSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([FlowsMeasuringUtilityTest]))
-
+    nightSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([FluidAuxiliaryUtilitiesTest]))
```

Comment:
those tests should run super fast no? => smallSuite

Reply:
Yes. These are super fast, but as we are tight on time in the small suite and these can be considered kind of secondary utilities I thought to give priority to the already existent ones. Makes sense for you?